source/admin_manual/list_of_events.rst

```diff
@@ -1113,6 +1113,29 @@ IMAP Command
    .. Note:: This event is currently not sent for pre-login IMAP commands.
 
 
+IMAP ID Command
+^^^^^^^^^^^^^^^
+
+.. dovecot_core:event:: imap_id_received
+   :added: v2.4.0;v3.0.0
+   :inherit: imap_client
+
+   :field cmd_name: Name of the command.
+   :field id_param_<param>: Received parameters. The event name is the lowercase
+                            parameter key prefixed with ``id_param_``, the value
+                            is the parameter value.
+   :field id_invalid<num>: Each key that contains invalid characters are
+                           enumerated starting with 1. Valid characters are
+                           latin alphabetic characters (= ``a`` .. ``z``),
```

Comment:
Should also document that the value contains the original key including invalid characters, followed by the value.